### PR TITLE
add lots of helper reveal widgets

### PR DESCRIPTION
**Pull request by loumoore**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1989,21 +1997,76 @@ en:
...
+          other_credits_itemized_deductions: Itemized deductions
+          other_credits_qcos_html: <a href="https://azdor.gov/tax-credits/credits-contributions-qcos-and-qfcos" target="_blank" rel="noopener nofollow">Credit for Contributions to Qualifying Charitable Organizations</a>
+          other_credits_qfcos_html: <a href="https://azdor.gov/tax-credits/credits-contributions-qcos-and-qfcos" target="_blank" rel="noopener nofollow">Credit for Contributions to Qualifying Foster Care Charitable Organization</a>
+          property_tax_credit_age: Be 65 or older; or receive Supplemental Security Income
+          property_tax_credit_heading_html: '<a href="https://azdor.gov/individuals/income-tax-filing-assistance/tax-credits" target="_blank" rel="noopener nofollow">Property Tax Credit</a>. To qualify for this credit you must:
+
+            '
+          property_tax_credit_income: have total household income less than $3,751 if you live alone; or total household income less than $5,501 if you live with others; and
+          property_tax_credit_paid: have paid property taxes or rent on a home in Arizona during the year
```
Comment (mpidcock) [inline config/locales/en.yml:2025]: This seemed off to me and I looked in the story, these bullet points should actually be capitalized

```suggestion
          property_tax_credit_income: Have total household income less than $3,751 if you live alone; or total household income less than $5,501 if you live with others; and
          property_tax_credit_paid: Have paid property taxes or rent on a home in Arizona during the year
```

**Review by mpidcock — CHANGES_REQUESTED** [review]
Nice work but I think the links should be updated before merging

**Comment by loumoore** [comment]
thanks martha! the pages to link to didn't exist yet, i think the FAQ pages are in progress. but i'll take a look again today.

File: config/locales/en.yml (hunk 1 above)
Comment (loumoore) [reply config/locales/en.yml:2025]: hmm yeah...it's lower case in the content doc which is what i was going off of, thought that was supposed to be the source of truth for copy

Comment (mpidcock) [reply config/locales/en.yml:2025]: oh gosh, idk. here's the figma screenshot from the story. i guess don't change it for now and if someone really cares we'll fix it later.

image

File: config/locales/en.yml [hunk 2]
```diff
@@ -1848,12 +1848,12 @@ en:
...
           charitable_noncash_html: 'Enter the <strong>total amount of non-cash contributions</strong> made in 2023 (example: the fair market value of donated items). This cannot exceed $500.'
           learn_more_content_html: |
-            Charitable contributions include cash and non-cash donations to qualifying organizations.<br><br>
-            Qualifying organizations include nonprofit groups that are religious, charitable, educational, scientific, or literary in purpose, or that work to prevent cruelty to children or animals. Check here to see if an organization qualifies.<br><br>
-            Do not include:<br>
+            Charitable contributions include <strong>cash</strong> and <strong>non-cash</strong> donations to qualifying organizations.<br><br>
+            Qualifying organizations include nonprofit groups that are religious, charitable, educational, scientific, or literary in purpose, or that work to prevent cruelty to children or animals. Check <a href="https://www.irs.gov/TEOS" target="_blank" rel="noopener nofollow">here</a> to see if an organization qualifies.<br><br>
+            Do <strong>not</strong> include:<br>
```
[the remark this answers is not in the document]
Comment (loumoore) [reply config/locales/en.yml:1853]: i believe the reason one might do that is cuz ": " is recognized as the key separator in yaml, but there are numerous options for handling that. in this case we're using the pipe separator so the entire string is interpreted as literal. you can also just enclose it in quotes. either way, i personally prefer the clarity of seeing the symbol if it doesn't need to be represented as an html entity.

[the remark this answers is not in the document]
Comment (mpidcock) [reply config/locales/en.yml:1853]: I realized that after I left this comment, makes sense. appreciate the additional info

**Review by mpidcock — APPROVED** [review]

File: config/locales/en.yml [hunk 3]
```diff
@@ -1930,24 +1930,26 @@ en:
...
-            at <strong>GetYourRefund.</strong>
-          get_started_gyr: Get started with GetYourRefund by...
+          file_with_vita: You can still file your state taxes for free with help from IRS-certified VITA volunteers in Arizona. Note that this option may take 2-3 weeks to complete.
+          get_started_with_vita: Get started with VITA now
           ineligible_reason:
             has_irc_125_code: your W2 lists IRC 125 benefit plan amounts.
             has_yonkers_income: you lived or earned income in Yonkers in 2023.
             married_filing_separately: you are filing “married filing separately”.
+          learn_more_content_how: IRS-certified VITA volunteers will use this information to prepare and e-file your state return. This entire process can take 2-3 weeks to complete.
+          learn_more_content_link_html: Learn more <a href="TODO">here</a>.
```
[the remark this answers is not in the document]
Comment (loumoore) [reply config/locales/en.yml:1940]: ah you're right good find. that page doesn't have final copy yet, it's all placeholders, but it appears to be the correct destination so I'm updating the TODO links to use it. looks like the content is still being iterated on. thanks!

**Pull request merged**